frontend/src/components/messages/NewRequestConfirmationDialog.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <AppDialog v-model="isDisplayed" :title="dialogTitle" persistent max-width="40%" @closeDialog="closeDialog">
+    <template #content>
+      <v-row>
+        <v-col align="center">
+          <p class="pt-4 text-h5">Your message has been submitted.</p>
+          <p class="pt-4 text-h5">Reference: {{ referenceNumber }}</p>
+          <p class="pt-4 text-h5">Typical response times are 3-5 business days.</p>
+        </v-col>
+      </v-row>
+    </template>
+    <template #button>
+      <v-row justify="space-around">
+        <AppButton id="return-home-button" :isPrimary="false" size="large" width="200px" @click="returnToHome()">Return to home</AppButton>
+        <AppButton id="view-messages-button" size="large" width="200px" @click="viewMessages()">View messages</AppButton>
+      </v-row>
+    </template>
+  </AppDialog>
+</template>
+
+<script>
+import AppButton from '../ui/AppButton.vue'
+import AppDialog from '../ui/AppDialog.vue'
+import { PATHS } from '@/utils/constants'
+
+export default {
+  name: 'NewRequestConfirmationDialog',
+  components: { AppButton, AppDialog },
+  props: {
+    showNewRequestConfirmationDialog: {
```

Comment:
I think within the context of a component "show" is probably sufficient. When you bind the prop from the parent component, you can use a more verbose property name. E.g.  :show="showNewRequestDialog".